core/src/main/java/org/apache/carbondata/core/metadata/datatype/Field.java

```diff
@@ -193,6 +192,7 @@ public void setParent(String parent) {
   }
 
   public String getStoreType() {
+    String storeType = "columnar";
```

Comment:
Q1: why this change? why not mark the string as static instead?
Q2: this method is never used

Reply:
removed this method